Dockerfile

```diff
@@ -1,18 +1,11 @@
-FROM gliderlabs/alpine:edge
-
-RUN apk --update add \
-  ca-certificates \
-  bash \
-  jq \
-  curl \
-  git
-
-# can't `git pull` unless we set these
-RUN git config --global user.email "git@localhost" && \
-    git config --global user.name "git"
+FROM alpine
 
+ADD assets/ /opt/resource/
 ADD scripts/install_git_lfs.sh install_git_lfs.sh
-RUN ./install_git_lfs.sh
 
-ADD assets/ /opt/resource/
-RUN chmod +x /opt/resource/*
+RUN set -ex && \
```

Comment:
I don't really see the point of keeping everything in a single layer. The resources will change every time while the apk packages won't. No need to recalculate that layer every time...